c++/src/capnp/compiler/capnpc-c++.c++

```diff
@@ -2445,6 +2445,8 @@ private:
           "  explicit Client(::kj::Own< ::capnp::ClientHook>&& hook);\n"
           "  template <typename _t, typename = ::kj::EnableIf< ::kj::canConvert<_t*, Server*>()>>\n"
           "  Client(::kj::Own<_t>&& server);\n"
+          "  template <typename _t, typename = ::kj::EnableIf< ::kj::canConvert<_t*, Server*>()>>\n"
+          "  Client(::kj::Rc<_t>&& server);\n"
```

Comment:
This code generator change should require regenerating bootstraps.

Looks like the test for stale bootstraps is an ekam-rule still, so we're missing it in CI, oops: `c++/src/capnp/bootstrap-test.ekam-rule`

Reply:
I'll take a look at fixing it.